indoor-map/demo.js

```diff
@@ -113,8 +109,5 @@ var behavior = new H.mapevents.Behavior(new H.mapevents.MapEvents(map));
 // Step 4: create the default UI component, for displaying bubbles
 var ui = H.ui.UI.createDefault(map, defaultLayers);
 
-// Indoor Maps provider interacts with a tile layer to visualize and control the Indoor Map
-const venuesProvider = new H.venues.Provider();
-
 // Step 5: add the Indoor Map
-addVenueToMap(map);
+addVenueToMap(map, platform);
```

Comment:
no need to pass the platform instance, it's already visible from inside the scope of `addVenueToMap`

```suggestion
addVenueToMap(map);
```

Reply:
Interestingly, in the user feedback, which this ticket is based on, there's a remark saying: 

"Variable `platform` is a global variable used in function `addVenueToMap. It is declared below the function but this is easily missed. I was trying to copy the function to an existing app and so got confused why this was not working."

Passing the platform instance in this case is a recommendation I received from Andrii Heonia. 

Or is there still no need to pass the platform instance here?

Reply:
I discussed it with Andrii, this is fine